v21.1/delete-data.md

```diff
@@ -78,38 +122,99 @@ try (Connection connection = ds.getConnection()) {
 ~~~ python
 # conn is a psycopg2 connection
 
+tsOne = '2021-03-17 14:00:00'
+tsTwo = '2021-03-17 15:00:00'
+
 with conn.cursor() as cur:
-    cur.execute("DELETE FROM accounts WHERE id = 1",
+    cur.execute(
+        "DELETE FROM vehicle_location_histories WHERE timestamp BETWEEN %s AND %s", (tsOne, tsTwo))
 conn.commit()
 ~~~
 
-{% include {{page.version.version}}/app/for-a-complete-example-python.md %}
+</section>
+
+{{site.data.alerts.callout_success}}
+If the `WHERE` clause evaluates to `TRUE` for a large number of rows (i.e., tens of thousands of rows), use a [batch-delete loop](bulk-delete-data.html#batch-delete-on-an-indexed-column) instead of executing a simple `DELETE` query.
+{{site.data.alerts.end}}
+
+#### Delete rows filtered on a unique column
+
+Suppose that you want to delete the promo code data for a specific set of codes. To delete the rows in the `promo_codes` table where the `code` matches a string in a set of string values:
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="sql">SQL</button>
+  <button class="filter-button" data-scope="go">Go (lib/pq)</button>
+  <button class="filter-button" data-scope="java">Java (jdbc)</button>
+  <button class="filter-button" data-scope="python">Python (psycopg2)</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="sql">
+
+{% include copy-clipboard.html %}
+~~~ sql
+DELETE from promo_codes WHERE code IN ('0_explain_theory_something', '100_address_garden_certain', '1000_do_write_words');
+~~~
+
+For more information about how to use the built-in SQL client, see the [`cockroach sql`](cockroach-sql.html) reference docs.
 
 </section>
 
-## Delete multiple rows
+<section class="filter-content" markdown="1" data-scope="go">
 
-You can delete multiple rows from a table in several ways:
+{% include copy-clipboard.html %}
+~~~ go
+// tx is a *sql.Tx from "database/sql"
```

Comment:
we could also use a `*sql.DB` here, which matches the other examples that don't use an explicit transaction